include/common.sh

```diff
@@ -18,7 +18,9 @@ export BUILD_DIR="$ROOT_DIR"/build${CLUSTER_NAME}
 # Forces our build context
 [ -d "$BUILD_DIR" ] && pushd "$BUILD_DIR"
 
-export CHART_URL="${CHART_URL:-}"
+export CHART_URL="${CHART_URL:-}" # FIXME deprecated, used in SCF_CHART
+export SCF_CHART="${SCF_CHART:-CHART_URL}"
```

Comment:
```make[3]: Leaving directory '/home/travis/build/SUSE/catapult/modules/scf'

Makefile:176: recipe for target 'scf-chart' failed

make[2]: *** [scf-chart] Error 2

make[2]: Leaving directory '/home/travis/build/SUSE/catapult'

ASSERT:helm folder is present```

I think is caused because it should be: ` export SCF_CHART="${SCF_CHART:-$CHART_URL}"`

Reply:
toootally true! thanks! :D